bitsail-common/src/test/resources/fake-type-converter.yaml

```diff
@@ -8,7 +8,7 @@ engine.type.to.bitsail.type.converter:
   - source.type: date
     target.type: date.date
 
-  - source.type: long
```

Comment:
Is it necessary to modify this type?

Reply:
The logic of this modification is to verify `toTypeInformation.put(StringUtils.lowerCase(entry.getKey()), customTypeInfo);` works, we store `long` as key rather than `Long`.